docs/en/tutorials/customize_rpn.md

```diff
@@ -0,0 +1,174 @@
+# Customize RPN
+
+Region proposal network(RPN) is a submodule in [Faster R-CNN](https://arxiv.org/abs/1506.01497), which generates proposals for the second stage of Faster R-CNN. Most two stage detectors in MMDetection use [`RPNHead`](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/mmdet/models/dense_heads/rpn_head.py) to generate proposals as RPN. However, any single-stage detector can serve as a RPN since their bounding box predictions can also be regarded as region proposals and thus be refined in the R-CNN. Therefore, MMDetection v3.0 supports that. Here we give an example to illustrate the whole process.
+
+The main contents are as follows:
+
+1. How to make `FCOSHead` as a `RPNHead` in Faster R-CNN
+2. How to evaluate the quality of proposals
+3. How to train the customized Faster R-CNN with pre-trained FCOS
+
+## How to make `FCOSHead` as a `RPNHead` in Faster R-CNN
+
+To make `FCOSHead` as a `RPNHead` in Faster R-CNN, we should create a new config file named `configs/faster_rcnn/faster-rcnn_r50_fpn_fcos-rpn_1x_coco.py`, and replaced with the setting for `rpn_head` with the `bbox_head` setting from `configs/fcos/fcos_r50-caffe_fpn_gn-head_1x_coco.py`. Besides, we still use the neck setting of FCOS whose strides is `[8, 16, 32, 64, 128]`, and update `featmap_strides` of `bbox_roi_extractor` to `[8, 16, 32, 64, 128]`. To avoid loss goes NAN, we apply warmup during the first 1000 iterations instead of the first 500 iterations, which means that the lr increases more slowly. The config is as below
```

Comment:
```suggestion
To set `FCOSHead` as an `RPNHead` in Faster R-CNN, we should create a new config file named `configs/faster_rcnn/faster-rcnn_r50_fpn_fcos-rpn_1x_coco.py`, and replace the settings of `rpn_head` with the `bbox_head` settings from `configs/fcos/fcos_r50-caffe_fpn_gn-head_1x_coco.py`. Besides, we still use the neck setting of FCOS with strides of `[8, 16, 32, 64, 128]`, and update `featmap_strides` of `bbox_roi_extractor` to `[8, 16, 32, 64, 128]`. To avoid NAN loss, we apply warmup in the first 1000 iterations instead of the first 500 iterations, which means that the lr increases more slowly. The config is as follows:
```